SKILL.tsv

```diff
@@ -3975,60 +3975,60 @@ SKILL_20150414_003974	{memo X}Duration of [Install Distribution Table] Yogurt in
+SKILL_20150414_004004	$Gather Corpse: Shock
+SKILL_20150414_004005	$Enemies hit by [Gather Corpse] are affected by the [Shock] status effect for 10 seconds.
+SKILL_20150414_004006	$Revive: Duration
 SKILL_20150414_004007	{memo X}Duration of [Revive] increase by 10 seconds per attribute level.
-SKILL_20150414_004008	Palm Strike: Bleed
+SKILL_20150414_004008	$Palm Strike: Bleed
 SKILL_20150414_004009	{memo X}Enemy falls under [Bleed] for 8 seconds when attacked with [Palm Strike]. Duration of [Bleed] incerases by a seconds per level.
-SKILL_20150414_004010	Hand Knife: Destroy Armor
+SKILL_20150414_004010	$Hand Knife: Armor Break
 SKILL_20150414_004011	{memo X}Enemies attacked with [Hand Knife] have chance of falling under [Armor Break]. Chance increases by 5% per attribute level.
-SKILL_20150414_004012	1 inch Punch: Silence
-SKILL_20150414_004013	Enemies fall under [Silence] for 5 seconds when attacked with [1inch Punch]
-SKILL_20150414_004014	Arcane Energy: Additional Damage
+SKILL_20150414_004012	$1 Inch Punch: Silence
+SKILL_20150414_004013	$Enemies hit by [1 Inch Punch] will be affected by the [Silence] status effect for 5 seconds.
+SKILL_20150414_004014	$Arcane Energy: Additional Damage
 SKILL_20150414_004015	{memo X}Enemies nearby receive Holy damage when using [Arcane Energy]. Damage increases per attribute level.
-SKILL_20150414_004016	Chortasmata: Duration
+SKILL_20150414_004016	$Chortasmata: Duration
 SKILL_20150414_004017	{memo X}Duration of [Chortasmata] increases by a second per attribute level.
 SKILL_20150414_004018	{memo X}Indulgentia: Additional Damage
 SKILL_20150414_004019	{memo X}Enemies receive additional damage amounting to 10% of character's magic damage per attribute level when attacked with [Indulgentia].
-SKILL_20150414_004020	Discerning Evil : Increase nunber of targets
+SKILL_20150414_004020	$Discerning Evil: Increase Number of Targets
 SKILL_20150414_004021	{memo X}Target of [Discerning] increase by 1 per attribute level.
-SKILL_20150414_004022	Call of Deities: Fade
+SKILL_20150414_004022	$Call of Deities: Fade
 SKILL_20150414_004023	Character is not threatened by monster summoned with [Call of Deities]. (except pre-emptive attack types)
-SKILL_20150414_004024	Forecast: Duration
+SKILL_20150414_004024	$Forecast: Duration
 SKILL_20150414_004025	{memo X}Duration of [Forecast] increases by a seconds per attribute level.
-SKILL_20150414_004026	Install Perch: Range Increase
+SKILL_20150414_004026	$Install Perch: Increase Range
 SKILL_20150414_004027	{memo X}Range of maintaining perch in [Install Perch] increases by 20 per attribute level.
 SKILL_20150428_004028	{memo X}Magic Missile
-SKILL_20150428_004029	{#993399}{ol}[Magic]{/}{/}{nl}Use Magic bullets to attack enemy continuously. A new bullet is fired when shooting enemy under sleep.
+SKILL_20150428_004029	${#993399}{ol}[Magic]{/}{/}{nl}Attack enemies consecutively by using magic bullets. A new bullet is fired when attacking enemies under sleep.
```

Comment:
I'd reword that as  "An additional bullet is fired when attacking enemies under [Sleep] condition."